packages/thirdweb/package.json

```diff
@@ -162,17 +162,17 @@
     "@radix-ui/react-focus-scope": "1.0.4",
     "@radix-ui/react-icons": "1.3.0",
     "@radix-ui/react-tooltip": "1.0.7",
-    "@tanstack/react-query": "5.29.2",
-    "@walletconnect/ethereum-provider": "2.12.2",
-    "@walletconnect/sign-client": "^2.13.1",
+    "@tanstack/react-query": "5.45.0",
+    "@walletconnect/ethereum-provider": "2.13.3",
+    "@walletconnect/sign-client": "^2.13.3",
     "abitype": "1.0.0",
     "fast-text-encoding": "^1.0.6",
     "fuse.js": "7.0.0",
     "input-otp": "^1.2.4",
     "mipd": "0.0.7",
     "node-libs-browser": "2.2.1",
     "uqr": "0.1.2",
-    "viem": "2.13.7"
+    "viem": "2.14.0"
```

Comment:
need to test this and the wallet connect change - last time i updated WC it broke us xD